aws-android-sdk-kinesisvideo/src/main/java/com/amazonaws/kinesisvideo/producer/KinesisVideoFrame.java

```diff
@@ -69,6 +69,9 @@ public KinesisVideoFrame(int index, int flags, long decodingTs, long presentatio
         mPresentationTs = presentationTs;
         mDuration = duration;
         mData = requireNonNull(data);
+        // In some devices encoder would generate frames with more than 3 trailing zeros
+        // which is not allowed by AnnexB specification
+        removeTrailingZeros();
```

Comment:
@MushMal I'm not familiar with the AnnexB spec. Could trimming all the zeroes be an issue? Should there be a safety-maximum number of bytes trimmed?

Everything else looks to be in order. Approved on my side.

Reply:
@minbi the important termination condition is in the code - not to overrun the beginning. The rest of the validations will be performed in the lower layers. So, if we end up with no bytes, we will eventually get an error from the bottom layer. Similarly, if we end up with semantically incorrect Annex-B format, it will be caught in the native layer. I think adding extra validation for Annex-B format here is not needed.